src/frontend/Utils/filterUtilsFunctions.js

```diff
@@ -0,0 +1,72 @@
+const filterByTopics = (state, videos) => {
+  switch (state.topic) {
+    case "leadership":
+      return [...videos].filter((video) => video.categoryName === state.topic);
+    case "focus":
+      return [...videos].filter((video) => video.categoryName === state.topic);
+    case "mastery":
+      return [...videos].filter((video) => video.categoryName === state.topic);
+    case "creativity":
+      return [...videos].filter((video) => video.categoryName === state.topic);
+    case "code":
+      return [...videos].filter((video) => video.categoryName === state.topic);
+    case "computer":
+      return [...videos].filter((video) => video.categoryName === state.topic);
+    case "meditation":
+      return [...videos].filter((video) => video.categoryName === state.topic);
+    case "science":
+      return [...videos].filter((video) => video.categoryName === state.topic);
+    default:
+      return videos;
+  }
```

Comment:
There must be some alternative for this cases. You can think about it.

Reply:
yes @vijaytembugade, he can make it in a single line for all instead of repeating himself ```follow DRY```